src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
 :relative-path: ../../../
-include::{docdir}/variables.adoc[]
\ No newline at end of file
+include::{docdir}/variables.adoc[]
+
+In this part we will explain how to use the **Standard User Workflow** in the nominal case, i.e. using the default implementations of each parts of the workflow (For example, the default implementation to send the confirmation email use the _Mailjet_ API).
+
+Use the Standard User Workflow can be done in 3 parts :
+
+* Use the Standard User Workflow in your worker
+* Add properties to configure the Standard User Workflow
+* Create the client part
+
+== Bootstrap
+
+In this Quickstart, we will begin with a generated application using the CLI. So you can run :
+
+[source, console]
+----
+$ npm init @zetapush myApp
+----
+
+We also use the package `@zetapush/user-management` so you can install it with :
```

Comment:
We import the library `@zetapush/user-management` that provides the Standard User Workflow using NPM: